locksmith/__tests__/app.test.js

```diff
@@ -0,0 +1,110 @@
+const request = require('supertest')
+const app = require('../src/app')
+const Lock = require('../src/sequelize')
+
+beforeEach((done) => {
+  Lock.create({ name: 'Test Lock', address: '0xbac', owner: '0xdeadbed123' }).then(() => {
```

Comment:
Can you use a proper length address for the test? I think it's 24 characters. You can copy from the CreatorLock storybook stories.

Also, use a constant for it and then reference the constant in the tests, will make refactoring easier if lock format changes